pkg/manage/server.go

```diff
@@ -108,3 +116,20 @@ func (c *ServerConfig) DatastoreWriterURL() url.URL {
 	}
 	return u
 }
+
+// waitForShutdown blocks until the service exists.
+//
+// It listens on SIGINT and SIGTERM. If the signal is received for a second
+// time, the process is killed with statuscode 1.
+func waitForShutdown() {
+	sigint := make(chan os.Signal, 1)
+	// syscall.SIGTERM is not pressent on all plattforms. Since the autoupdate
```

Comment:
Fix typo pls. We do not have autoupdate here.

Reply:
done